packages/react/src/lib/useSoundPlayer.ts

```diff
@@ -195,11 +206,24 @@ export const useSoundPlayer = (props: {
     setFft(generateEmptyFft());
   }, []);
 
+  const muteEVI = useCallback(() => {
+    clearQueue();
```

Comment:
I'm not sure clearing the audio clip queue is what we want this feature to do - expected behavior would be similar to the mute button on your laptop keyboard where, that if the voice has several clips lined up, the clips would still play as usual, but silently. Clicking "unmute" would bring the loudness back up . Clearing the queue, on the other hand, deletes all of the assistant responses that have come in.

We're using audio context for audio playback, so creating a gain node seems like a potential path forward: https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createGain

Reply:
Ohh that makes sense! Thank you Yini, I will look into this!